sound/soc/sof/topology.c

```diff
@@ -473,143 +473,135 @@ static enum sof_comp_type find_process_comp_type(enum sof_ipc_process_type type)
 struct sof_topology_token {
 	u32 token;
 	u32 type;
-	int (*get_token)(void *elem, void *object, u32 offset, u32 size);
+	void (*load_token)(void *elem, void *object, u32 offset, u32 size);
```

Comment:
this is a bit heavy, could we instead keep the same name but change the return type to void?

in practice we never returned anything other than zero and never tested the result, so that's a simplification regardless of the IPC changes, no?

Reply:
The reason I changed the name is because I do need something that returns the value as well and if this is retained as a get without a return value, it would be quite confusing

Reply:
it's quite common that a read/get returns a status, but the actual value is handled as a function parameter. That allows for separation between error handling and actually returning a value.
If there is no error handling, a get with a void return type is perfectly fine in my book.